commanderbot_ext/jira/jira_cog.py

```diff
@@ -0,0 +1,91 @@
+import requests
+import discord
+from commanderbot_lib.logging import Logger, get_clogger
+from discord.ext.commands import Bot, Cog, command
+
+
+class JiraCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot: Bot = bot
+        self._log: Logger = get_clogger(self)
+
+        self.resolution_table = {
+            "https://bugs.mojang.com/rest/api/2/resolution/1": "Fixed", 
+            "https://bugs.mojang.com/rest/api/2/resolution/2": "Won't Fix", 
+            "https://bugs.mojang.com/rest/api/2/resolution/3": "Duplicate", 
+            "https://bugs.mojang.com/rest/api/2/resolution/4": "Incomplete", 
+            "https://bugs.mojang.com/rest/api/2/resolution/5": "Cannot Reproduce", 
+            "https://bugs.mojang.com/rest/api/2/resolution/6": "Works as Intended",
+            "https://bugs.mojang.com/rest/api/2/resolution/7": "Invalid",
+            "https://bugs.mojang.com/rest/api/2/resolution/10001": "Awaiting Response",
+            "https://bugs.mojang.com/rest/api/2/resolution/10003": "Done"
+        }
+
+        self.status_table = {
+            "https://bugs.mojang.com/rest/api/2/status/1": "Open",
+            "https://bugs.mojang.com/rest/api/2/status/3": "In Progress",
+            "https://bugs.mojang.com/rest/api/2/status/4": "Reopened",
+            "https://bugs.mojang.com/rest/api/2/status/5": "Resolved",
+            "https://bugs.mojang.com/rest/api/2/status/6": "Closed",
+            "https://bugs.mojang.com/rest/api/2/status/10200": "Postponed"
+        }
+
+    # TODO remove hardcoded status and resolve values and add config for JIRA URL and bug ID format
+    @command(aliases=["jira", "bug"])
```

Comment:
Should be:

```@command(name="jira", aliases=["bug"])```

Otherwise the `name` defaults to the function name `cmd_jira`.